### PR TITLE
Working

### PR DESCRIPTION
**Pull request by BSGreaves**

### CODE REVIEW

**Comment by BSGreaves** [comment]
@drdreed
WYSIWYG Ready for feedback

File: style.css [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+}
+
+input {
+  display: block;
+  margin: auto;
+  padding: 5px;
+  font-size: 1.5em;
+}
+
+/*#famousButton {
```
Comment (DevDreed) [inline style.css]: Delete unused CSS.

File: style.css [hunk 2]
```diff
@@ -0,0 +1,130 @@
+body {
+	font-family: Futura, "Trebuchet MS", Arial, sans-serif;
```
Comment (DevDreed) [inline style.css:2]: Fix formatting.

File: main.js [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+        parentElement = e.target.parentNode; //Makes sure I select the card!
+    }
+    parentElement.className += " dottedBorder"; //adds dotted border to person card
+    userInput.focus(); //adds focus to input field
+    userInput.addEventListener("keyup", function() { //pairs person's bio and input field
+        parentElement.children[4].innerText = userInput.value;
+    });
+    userInput.addEventListener("keypress", function(f) { //empties input field
+        if (f.keyCode === 13) {
+            userInput.value = "";
```
Comment (DevDreed) [inline main.js]: This is clearing the Bio field as well instead of just the input field.

**Review by DevDreed — CHANGES_REQUESTED** [review]
Please make the changes we discussed.  Also add a Readmed 😄

**Comment by BSGreaves** [comment]
Fixed up the functions and event listeners... now the bios are saveable. Implemented some flags to prevent bugs. Etc.

**Comment by BSGreaves** [comment]
@drdreed Fixed up the functions and event listeners... now the bios are saveable. Implemented some flags to prevent bugs. Etc.

**Pull request merged**